handbook/product/product_documents.md

```diff
@@ -0,0 +1,65 @@
+# Product documents
+
+A Product Document (PD)'s purpose is to communicate the high level product problem that needs to be solved. It is the one place where ongoing research and information can be aggregated around a particular problem statement. Product documents exist to encourage quick iteration, the ability to fail fast, and multiple efforts to be combined towards a singular problem statement.
+
+These documents were created out of a need to orient around a particular problem statement, and the realization that it sometimes takes multiple [RFCs](../communication/rfcs.md) to fully solve a problem. This helps to aggregate the motivations and learnings around a problem, so that individual RFCs can focus on a particular proposed solution.
+
+_All PDs are in a [public Google Drive folder](https://drive.google.com/drive/folders/1Wd-Xx2wNbFtSzeJwbZqMOxdbFDUFxlyR)._
+
+## PDs have similar properties to RFCs
+
+They are:
+
+- [Sequentially numbered](../communication/rfcs.md#rfcs-are-sequentially-numbered)
+- [Google Docs](../communication/rfcs.md#rfcs-are-google-docs)
+- [Public](../communication/rfcs.md#rfcs-are-public)
+- [Open to external contributions](../communication/rfcs.md#external-contributors)
+
+## Status
+
+Each PD has a status that is in the title of the PD (e.g. "PD 1 WIP: Title"). The author is responsible for keeping the status updated.
+
+| Status | Description |
+|-------|-------------|
+| WIP | The author is still drafting the PD and it is not ready for review. |
+| REVIEW | The PD is ready to be reviewed. The PD explicitly lists whose approvals are required and a requested timeline for those approvals. |
```

Comment:
I have been worried about the framing of "required approvals" for RFCs and think it would be more effective (and give authors more ownership) to think of it like "requested reviewers".

```suggestion
| REVIEW | The PD is ready to be reviewed. The PD explicitly lists whose reviews are requested and deadline for receiving those reviews. |
```

Reply:
Good catch - I started to clarify this in the language I have below about metadata, but missed it here.